docker/docker-compose-local.yml

```diff
@@ -22,6 +22,10 @@ services:
             - LOAD_EX=n
             - EXECUTOR=Local
             - AIRFLOW__WEBSERVER__WORKERS=1
+            - AIRFLOW__WEBSERVER__EXPOSE_CONFIG=true
```

Comment:
It's not mandatory but it's helpful to me to make sure things are working like I expect. And I think it's ok for dev setup !